Mathlib/Data/Finsupp/Defs.lean

```diff
@@ -4,7 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl, Scott Morrison
 
 ! This file was ported from Lean 3 source module data.finsupp.defs
-! leanprover-community/mathlib commit 8631e2d5ea77f6c13054d9151d82b83069680cb1
+! leanprover-community/mathlib commit 842328d9df7e96fd90fc424e115679c15fb23a71
```

Comment:
This file [still needs](https://leanprover-community.github.io/mathlib-port-status/file/data/finsupp/defs?range=8631e2d5ea77f6c13054d9151d82b83069680cb1..6623e6af705e97002a9054c1c05a980180276fc1) a forward-port of leanprover-community/mathlib#18277, which would be best as a standalone PR. It's possible there's nothing to port there.

Reply:
done in #2653